apps/dashboard/src/components/preview-subscriber-section.tsx

```diff
@@ -38,22 +39,23 @@ export function PreviewSubscriberSection({
           </div>
           {onClearPersisted && (
             <div className="mr-2">
-              <Button
+              <div
```

Comment:
fixed the issue with button under button in the dom tree

Reply:
Oh nice!